systemtest/src/test/java/io/strimzi/systemtest/operators/CustomResourceStatusST.java

```diff
@@ -449,16 +451,25 @@ void deployTestSpecificResources() throws Exception {
                             .withType(KafkaListenerType.INTERNAL)
                             .withTls(true)
                         .endGenericKafkaListener()
-                        .addNewGenericKafkaListener()
-                            .withName(Constants.EXTERNAL_LISTENER_DEFAULT_NAME)
-                            .withPort(9094)
-                            .withType(KafkaListenerType.NODEPORT)
-                            .withTls(false)
-                        .endGenericKafkaListener()
                     .endListeners()
                 .endKafka()
-            .endSpec()
-            .done();
+            .endSpec();
+
+        if (!Environment.isNamespaceRbacScope()) {
+            kafka.editSpec()
+                    .editKafka()
+                        .editListeners()
+                            .addNewGenericKafkaListener()
+                                .withName(Constants.EXTERNAL_LISTENER_DEFAULT_NAME)
+                                .withPort(9094)
+                                .withType(KafkaListenerType.NODEPORT)
+                                .withTls(false)
+                            .endGenericKafkaListener()
+                        .endListeners()
+                    .endKafka()
+                .endSpec();
+        }
```

Comment:
I wonder how well the status of the node port listener is really tested if this is all you need to get it passing. Because I guess it means there are no asserts for the status part related to the NodePort. So do we really need this? Should we just delete this and remove the `@Tag(NODEPORT_SUPPORTED)` annotation where applicable?

WDYT @Frawless?

Reply:
Or we can enhance the test for some additional checks, but I think to remove nodePort part is better since we take data from Kafka status in different tests.